core/server/master/src/main/java/alluxio/master/file/contexts/CreateFileContext.java

```diff
@@ -24,6 +26,7 @@
     extends CreatePathContext<CreateFilePOptions.Builder, CreateFileContext> {
 
   private boolean mCacheable;
+  private boolean mIsAsync;
```

Comment:
Instead of having an `isAsync` boolean, should we store the write type? This could be done in the Path context and also replace the other derived value (`isPersisted`)

Reply:
Added write type on `CreatePathContext`